buildSrc/subprojects/plugins/src/main/kotlin/gradlebuild/security.gradle.kts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package gradlebuild
+
+import java.net.URI
+
+val allowedSchemes = setOf("https")
+val insecureRepos = mutableSetOf<String>()
+
+allprojects {
+    configurations.all {
+        incoming.beforeResolve {
+            repositories.checkRepositories()
+        }
+    }
+    afterEvaluate {
+        extensions.findByType(PublishingExtension::class.java)?.run {
+            repositories.checkRepositories()
+        }
+    }
+}
+
+gradle.buildFinished {
+    if (!insecureRepos.isEmpty()) {
+        throw GradleException("This build used insecure repositories:\n" + insecureRepos.joinToString("\n") + "Make sure to use HTTPS")
```

Comment:
From the failing build, looks like you need an additional `\n` before that `Make sure ...`

Reply:
fixed